packages/headless/src/controllers/commerce/facets/core/headless-core-commerce-facet.ts

```diff
@@ -59,14 +59,14 @@ export interface CoreCommerceFacetProps {
 export interface CoreCommerceFacetOptions {
   facetId: string;
   toggleSelectActionCreator: ActionCreatorWithPreparedPayload<
-    [payload: AnyToggleFacetValueActionCreatorPayload],
+    [payload: any], // eslint-disable-line @typescript-eslint/no-explicit-any
```

Comment:
TypeScript started screaming when I passed the category facet action creator so I took the easy way out on this one...

Reply:
We'll have to figure this one out, tho it can wait post basecamp.
Please add a todo/tech debt Jira for this.

Having Typescript angry in the prod code is dangerous 😅

Reply:
I managed to get rid of the any's!